docs/standard/base-types/how-to-verify-that-strings-are-in-valid-email-format.md

```diff
@@ -22,7 +22,10 @@ author: "rpetrusha"
 ms.author: "ronpet"
 ---
 # How to: Verify that Strings Are in Valid Email Format
-The following example uses a regular expression to verify that a string is in valid email format.  
+The following example is more complex than previous examples and uses a regular expression to verify that a string is in valid email format.  
+
+> [!NOTE]
+>  Use the <xref:System.Net.Mail.MailAddress?displayProperty=nameWithType> class to check if a string is in valid email address format. To do that, pass the email address string to the <xref:System.Net.Mail.MailAddress.%23ctor%28System.String%29?displayProperty=nameWithType> class constructor that throws a <xref:System.FormatException> if the string has unrecongnized format.  
```

Comment:
constructor that throws --> constructor, which throws
has --> has an
unrecongnized --> unrecognized